nengo/simulator.py

```diff
@@ -55,6 +55,12 @@ def __len__(self):
 class Simulator(object):
     """Reference simulator for Nengo models."""
 
+    unsupported = []
+    """Defines features unsupported by a simulator. The format is a list of
```

Comment:
Right, that's a good point. I made https://github.com/nengo/nengo.github.io/issues/7 to track the backend dev doc; for now turning into a comment makes sense.